packages/components/src/menu/radio-item.tsx

```diff
@@ -51,7 +51,7 @@ export const MenuRadioItem = forwardRef<
 				// Override some ariakit inline styles
 				style={ { width: 'auto', height: 'auto' } }
 			>
-				<Icon icon={ radioCheck } size={ 24 } />
+				{ checkedPrefix || <Icon icon={ radioCheck } size={ 24 } /> }
```

Comment:
I'm jumping in pretty blind here, so happy to change anything according to what's proper and what's not 😄